tools/VagrantSetup/Vagrantfile_nightly

```diff
@@ -14,7 +14,7 @@ Vagrant.configure(2) do |config|
     inline: "docker stop lanraragi || true && docker rm lanraragi || true"
   config.vm.provision "docker" do |d|
     d.pull_images "difegue/lanraragi:nightly"
-    d.run "difegue/lanraragi:nightly",
+    d.run "difegue/lanraragi:nightly:nightly",
```

Comment:
```suggestion
    d.run "difegue/lanraragi:nightly",
```

Reply:
You said "It's probably better to remove it altogether here so we don't have double :nightly in the following run command."

In the previous comment. But then added double nightly here, is this intentional?

Reply:
Two nighty's wasn't needed anyway, I made a mistake with git. I saw it was only on there once on the master branch vagrant file. But yeah, it's not necessary in the d.pull.

Reply:
That was actually unintentional, made a mistake with git and was looking at the master branch vagrantfile. oops.